system/HTTP/Response.php

```diff
@@ -1036,10 +1057,29 @@ protected function sendCookies()
 
 		foreach ($this->cookies as $params)
 		{
-			// PHP cannot unpack array with string keys
-			$params = array_values($params);
+			if (PHP_VERSION_ID < 70300)
+			{
+				// For PHP 7.2 we need to use the hacky method of setting SameSite in the path
+				if (isset($params['samesite']) && in_array(strtolower($params['samesite']), ['none', 'lax', 'strict']))
```

Comment:
I think we can manage without this check as we already handle it through the constructor.